plugins/common/parallel/parallel_test.go

```diff
@@ -31,7 +31,7 @@ func TestOrderedJobsStayOrdered(t *testing.T) {
 	p.Stop()
 
 	i := 0
-	require.Len(t, acc.Metrics, 20000, fmt.Sprintf("expected 20k metrics but got %d", len(acc.GetTelegrafMetrics())))
+	require.Len(t, acc.Metrics, 20000, "expected 20k metrics but got %d", len(acc.GetTelegrafMetrics()))
```

Comment:
I think in this case you need `Lenf`, don't you? In any case, I would just remove the message at all as that's what `Len` will output anyway.

Reply:
Message removed.